test/sensor_simulator/flow.cpp

```diff
@@ -15,7 +15,6 @@ Flow::~Flow()
 
 void Flow::send(uint64_t time)
 {
-	_flow_data.dt = static_cast<float>(time - _time_last_data_sent) * 1e-6f;
```

Comment:
why are you removing this?

Reply:
Because this is the "integration timespan" of the optical flow sensor and not the publishing rate. My actual issue was that the first sample had a really large dt (a few seconds).

Reply:
Ok. But then we should set it to 1 / sensor_rate